mne/io/base.py

```diff
@@ -938,9 +939,10 @@ def resample(self, sfreq, npad=100, window='boxcar',
                      speed up computations (e.g., projection calculation) when
                      precise timing of events is not required, as downsampling
                      raw data effectively jitters trigger timings. It is
-                     generally recommended not to epoch downsampled data,
-                     but instead epoch and then downsample, as epoching
-                     downsampled data jitters triggers. See here for an
+                     generally recommended to find events using the original
+                     data and jointly resample the raw and events using the
+                     'events' parameter. Alternatively, resampling can be
+                     performed on the epochs instead. See here for an
```

Comment:
Please restore the original warning suggesting use of `epochs` resampling over that of `raw`. Even when resampling the events there will be effective jitter.